### PR TITLE
Add issue & pr templates

### PR DESCRIPTION
**Pull request by dwijnand**
See https://github.com/blog/2111-issue-and-pull-request-templates

### CODE REVIEW

**Comment by dwijnand** [comment]
See:
- https://github.com/dwijnand/sbt/issues/new to see the issue tempalte and
- https://github.com/dwijnand/sbt/compare/1.0.x...dwijnand:upgrade-sbt-libman?expand=1 to see the pull request template

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(Strongly consider adding notes for this change)
```
Comment (eed3si9n) [inline PULL_REQUEST_TEMPLATE.md]: Maybe point to a document explaining what a note is and/or we should encourage ppl to document each commits and squash them appropriately.

**Comment by dwijnand** [comment]
@eed3si9n I added a small "Clean history" section to the Contributing guide, and made the call-to-action in the template stronger. It's only a small chance, so feel absolutely free to comment/review if you're looking for more. I've embraced the fact I'm bad (and slow) at documentation :)

**Comment by eed3si9n** [comment]
LGTM

**Comment by dwijnand** [comment]
Thank you.

**Pull request merged**